cypress/integration/line-chart.spec.js

```diff
@@ -0,0 +1,146 @@
+/// <reference types="Cypress" />
+
+context("Line Chart", () => {
+
+  const header = () => cy.get(".cube-header-bar");
+  const timeFilter = () => cy.get(".filter-tile .filter:first");
+  const lineChart = () => cy.get(".measure-line-chart");
+  const highlighter = () => lineChart().find(".highlighter");
+  const highlighterFrame = () => highlighter().find(".frame");
+  const highlightModal = () => cy.get(".highlight-modal");
+  const acceptHighlight = () => highlightModal().find(".accept");
+  const dropHighlight = () => highlightModal().find(".drop");
+
+  describe("Highlight", () => {
+
+    const urls = {
+      timeSplit: "http://localhost:9090/#wiki/4/N4IgbglgzgrghgGwgLzgFwgewHYgFwhLYCmAtAMYAWcATmiADQgYC2xyOx+IAomuQHoAqgBUAwoxAAzCAjTEaUfAG1QaAJ4AHLgVZcmNYlO57JegAoKsAEyV5VIazBrosuAuYCMAEUlR5mviknkwa2twI6MT+IAC+ALoJTFCaSGh2DmE6zBBskta5xNhQbiaFkgDmLtgwkTQQGtzmIp4AEn6YdPighsa65aFa2QVsxaVMBYbkGDjccFDkRQXYFXFMSCwN+DUICEkgUFbRKj1GCkWLc9bWxNaSUp0s6N3MQ9w3UnC19ExgiDDZNavcIENjzZxcfaaCDYEjWbyFMY4DIgKhwWHEBCSbBwNgpOCXJjQABKmAARph6PEmNCMdYAMqdegEODXW6SBDECpLba1PZAA"
+    };
+
+    const assertHighlighterPosition = (expectedLeft, expectedWidth) =>
+      highlighterFrame().should($frame => {
+        const frame = $frame.get(0);
+        expect(frame.offsetLeft).to.eq(expectedLeft);
+        expect(frame.offsetWidth).to.eq(expectedWidth);
+      });
+
+    beforeEach(() => {
+      cy.visit(urls.timeSplit);
+    });
+
+    describe("Highlight selection with click", () => {
+      beforeEach(() => {
+        lineChart().click(100, 200);
+      });
+
+      describe("selecting highlight", () => {
+        it("should show highlight modal", () => {
+          highlightModal().should("exist");
+        });
+
+        it("should show time period on highlight modal", () => {
+          highlightModal().should("contain", "12 Sep 2015 3:00 - 12 Sep 2015 4:00");
```

Comment:
A little bit fragile but I do not have better idea for assertion

Reply:
Remember that click targets are bigger than 1px, so it quite stable.